ChurchCRM/CRM
# Clear People and Families

#### What's this PR do?
- Updates the CVS Import Reset Member Data
- Adds Rest API to clear people, families, images and anything else that holds member people
- Switch Clear from Inline to AJAX Call

#### What Issues does it Close?

Closes  #3984

#### What are the relevant tickets?

#### How should this be manually tested?
- Import data (CVS, or Demo data restore
- Go to CVS Import Click clear data
- Ensure data is gone and you are still logged in 

#### Questions:
- Is there a related website / article to substantiate / explain this change?

- Does the development wiki need an update?

- Does the user documentation wiki need an update?

- Does this add new dependencies?

- Does this need to add new data to the demo database
